src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -99,16 +218,19 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityProps> = ({
     )
     if (!_UserlpData) return
     setUserlpData(_UserlpData)
+    const runeAddress = _UserlpData?.runeAddress
```

Comment:
Wrong casing in `_UserlpData` on l. 216?

Reply:
Nice catch: https://github.com/shapeshift/web/pull/6175/commits/c13578dbbaf318dd43792431b655e7f7601d26a5